Index.html

```diff
@@ -52,10 +52,182 @@ <h1>
+                <li>javascript</li>
+            </ul>
+            <button>See Project</button>
+        </div>
+     </article>
+
+     <article class="portfolio-work">
+        <img class="work-image" src="./assets/Snapshoot Portfolio 3.svg "alt="">
+    <div class="work-description">
+        <h1 class="work-title">Tonic</h1>
+        <ul class="job-description">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+        </ul>
+        <p>Tonic</p>
+        <ul class="work-technologie">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+        </ul>
+       <div> 
+        <button>See Project</button>
+    </div>
+    </div>
+ </article>
+ <article class="portfolio-work">   
+    <img class="work-image" src="./assets/Snapshoot Portfolio 4.svg" alt="">
+<div class="work-description">
+    <h1 class="work-title">Multi-Post <br> Stories</h1>
+    <ul class="job-description">
+        <li>CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="work-technologie">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+    <button class="button-btn">See Project</button>
+</div>
+</article>
+</section>
+
+<section id="portfolio">
+    <article class="portfolio-work">
+        <div>
+            <h1>About <br> Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+         </div>
+         <div>
+            <a href="" class="connect">
+                Let's Connect
+            </a>
+            <ul class="work-technologie">
+                <li>
+                    <a href="">
+                        <img src="./assets/Github icon.svg" alt="Github">
+                    </a>
+                </li>
+                    <li class="icons">
+                    <a href="">
+                        <img src="./assets/Linkedin icon.svg" alt="Linkedin">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/Angelist icon.svg" alt="Angelist">
+                    </a>
+                </li>
+                <li class="icons">
+                    <a href="">
+                        <img src="./assets/Twitter icon.svg" alt="Twitter">
+                    </a>
+                </li>
+                <li class="icons">
+                    <a href="">
+                        <img src="./assets/Medium icon.svg" alt="Medium">
+                    </a>
+                </li>
+            </ul>
+        <div class="button">Get My Resume</div>
+         </div>
```

Comment:
- Kindly, "Get my resume" is a link `a` or `button` tag, [see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=C0vU075hnDfZLPwM-0)👌👌 for semantic reasons because it may need interactions 👍